mulesoft-ai-chain/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,57 @@
+= MuleSoft AI Chain Connector 1.0 - Mule 4
+
+Anypoint Connector for MuleSoft AI Chain (MuleSoft AI Chain Connector) helps developers design, build and manage AI agents within Anypoint Platform. It provides the tools and support to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.
```

Comment:
"Anypoint Connector for MuleSoft AI Chain (MuleSoft AI Chain Connector)"

This reads oddly - the name should be the MuleSoft AI Chain Connector

Reply:
@rstastny9 The official naming convention for all MuleSoft connectors is Anypoint Connector for _____, then the short name in (). The only time we use the long name is in the first instance. Everywhere else it's just called MuleSoft AI Chain Connector.